### PR TITLE
Various fixes

### PR DESCRIPTION
**Pull request by fosslinux**

### CODE REVIEW

File: sysa/helpers.sh [hunk 1]
```diff
@@ -171,36 +191,31 @@ build() {
...
-                tar -xzf "/sources/${i}"
-                ;;
+            *.tar.gz) tar -xzf "${DISTFILES}/${i}" ;;
             *.tar.bz2)
                 # Initial bzip2 built against meslibc has broken pipes
-                bzip2 --decompress --keep "/sources/${i}"
-                tar -xf "/sources/${i%.bz2}"
-                rm "/sources/${i%.bz2}"
-                ;;
+                bzip2 -dc "${DISTFILES}/${i}" | tar -xf - ;;
```
Comment (fosslinux) [inline sysa/helpers.sh:216]: I'm not sure whether to simplify this or the bzip2 hack...

File: sysa/helpers.sh [hunk 2]
```diff
@@ -297,9 +318,8 @@ src_apply() {
                 rm -f "/${file}" >/dev/null 2>&1 || true
             done < /tmp/filelist.txt
         fi
-        bzip2 --decompress --keep "/usr/src/repo/${pkg}_${revision}.tar.bz2"
-        tar -C / -xpf "/usr/src/repo/${pkg}_${revision}.tar"
-        rm "/usr/src/repo/${pkg}_${revision}.tar"
+        bzip2 -dc "/usr/src/repo/${pkg}_${revision}.tar.bz2" | \
+            tar -C / -xpf -
```
Comment (fosslinux) [inline sysa/helpers.sh:322]: Not sure whether to simplify this and bzip2 hack..

Comment (stikonas) [reply sysa/helpers.sh:322]: Hmm, not sure whether we want to use pipe or keep old two step process. I guess I don't mind either

**Review by stikonas — APPROVED** [review]
Looks alright, other than that minor comment.

**Pull request merged**